.travis.yml

```diff
@@ -9,7 +9,7 @@ python:
 install:
     - pip install tox coveralls coverage
 script:
-    - tox -e py
+    - tox -e py,isort
```

Comment:
The reason is that this package contains tons of flake8 violations which I did not yet had want to fix for this PR.